safe/impact_function/impact_function.py

```diff
@@ -18,7 +18,8 @@
     QgsCoordinateReferenceSystem,
     QgsRectangle,
     QgsVectorLayer,
-    QGis
+    QGis,
+    QgsMapLayer
```

Comment:
Small trick, but I always add the comma, even if it's the last one. You have one line less in your github PR and in git blame for the next time.

Reply:
Done.